README.md

```diff
@@ -1,40 +1,146 @@
+* Hold the special Windows key on your keyboard and press the "X" key. Choose "Command Prompt" from the pop-up menu.
+* Hold the Windows key and press the "R" key to get a "Run" window. Type "cmd" in the box, and click the OK key.
+ןכ
+![image](https://github.com/TechGirlsTribe1/introduction_to_python/assets/6695760/17ba7471-8452-4a23-b96f-d1097655d811)
+
+Try it now on your computer and see what happens!
+
+**If you have an Apple laptop:**
+
+- Open the Terminal application (It is already installed by default on an Apple laptop)
+
+What is Terminal?
+The Terminal application on a Mac is a command-line interface that allows users to interact with the operating system and execute commands using text-based input.
+
+**Now, On Terminal if you have an Apple laptop, or on the Command Prompt if you have a Windows:**
+
+- Type  `python3 --version`
+
+With this command, we're checking if Python was installed, and if it was, then what is the version of Python which was installed - this is the best indiciation whether Python was installed properly.
 
-Can you run the commands below without errors?
-- Open terminal 
-- Execute ``python3 --version``
 Do you see the output?  
 >Python 3.N.N
-- Execute `python3 -m pip --version`
+- Type `python3 -m pip --version`
+
+With this command, we're checking if PIP was installed, and if it was, then what is the version of PIP which was installed - this is the best indiciation whether PIP was installed properly.
+
 Do you see the output?  
 >pip x.x.x from ....
 
 If you see the outputs above, then you have successfully installed Python! It is time to install Jupyter Notebooks!
 
 ### Install Jupyter Notebook
 
-https://jupyter.org/install
+What is Jupyter Notebook? yet another good question,
 
->pip install notebook
+Jupyter Notebook is a web-based tool where you can write and run code in a variety of languages (like Python, R, and Julia), visualize data, and create shareable documents. It's great for data cleaning, data visualization, prototyping, and sharing results.
+We will use Jupyter Notebook in the first part of the workshop, to run and test basic Python commands.
 
+Run this command which will use PIP to install the Jupyter Notebook:
+- Type `pip install notebook`
+
+More info can be found on: https://jupyter.org/install
 
 # Test it
 
-Start the notebook server from the command line:
-`jupyter notebook`
-You should see the notebook open in your browser.
+Let's test if the Jupyter Notebook was indeed installed.
+
+**In order to start the Notebook, on Terminal if you have an Apple laptop, or on the Command Prompt if you have a Windows:**
+
+- Type `jupyter notebook`
+  
+You should see the web page of the Notebook open in your browser.
+
+
+That's it, you're all set and ready for the workshop! good job!
```

Comment:
Shall we add a picture example here to show how it looks like on the working state?